### PR TITLE
#16678987 User should be able to sign out

### PR DESCRIPTION
**Pull request by manzif**
#### What does this PR do?

 - User should be able to sign out

#### Description of Task to be completed?

- Blacklist tokens used
- Drop tokens that are no longer valid or have expired. 
- Check if the tokens passed in the header are blacklisted or not.

#### How should this be manually tested?

- When you login in the system and get token, Whenever you want to log out you click on log out and It will use the token to log you out

#### Any background context you want to provide?

- N/A

#### What are the relevant pivotal tracker stories?

- [#166789871](https://www.pivotaltracker.com/story/show/166789871)

#### Screenshots (if appropriate)

image

image

#### Questions:

- N/A

### CODE REVIEW

File: src/db/migrations/20190704064417-create-blacklist-tokens.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
+    });
+  },
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.dropTable('BlacklistTokens');
+  }
+};
```
Comment (Dom58) [inline src/db/migrations/20190704064417-create-blacklist-tokens.js]: Leave new empty line

**Review by j4l13n — COMMENTED** [review]
Why did you change the expiration of token to 1 min?

**Review by Emile-Nsengimana — APPROVED** [review]
Why

**Review by adafia — COMMENTED** [review]
Please check the title of your commit message, remove the colon ':' at the beginning.

**Comment by manzif** [comment]
> Why did you change the expiration of token to 1 min?

In the demo, the TTL told me to put it 1 min

**Comment by manzif** [comment]
> Why

Why what??

**Comment by manzif** [comment]
> Please check the title of your commit message, remove the colon ':' at the beginning.



> Please check the title of your commit message, remove the colon ':' at the beginning.

Resolved

**Review by adafia — APPROVED** [review]

**Review by j4l13n — APPROVED** [review]

**Pull request merged**